cmd/kubeadm/app/phases/addons/manifests.go

```diff
@@ -68,7 +68,7 @@ spec:
     spec:
       containers:
       - name: kube-proxy
-        image: {{ .Image }}
+        image: {{ if .ImageOverride }}{{ .ImageOverride }}{{ else }}{{ .ImageRepository }}/kube-proxy-{{ .Arch }}:{{ .Version }}{{ end }}
```

Comment:
No need to support .ImageOverride here.
This will get it's own, overridable `.Image` field in the config file as we go. It shouldn't be tied to the hyperkube image as it's an addon. This was an oversight from earlier.